### PR TITLE
Git guide improvements

### PR DESCRIPTION
**Pull request by s-laugh**
Adding more detailed steps and links for post migration.

For topic #1

### CODE REVIEW

File: guides/tfvc-to-git.md [hunk 1]
```diff
@@ -6,9 +6,33 @@
...
+   ```bash
+   git tfs quick-clone "http://tfs.intra.dmz:8080/tfs/projectcollection" "$/EWS-SWE" .
+   ```
+1. Push to remote repository
+   ```bash
+   git remote add origin "http://tfs.intra.dmz:8080/tfs/projectcollection/EWS-SWE/_git/EWS-SWE-Git"
+   git push --all origin
+   ```
+
+### Looking forward & finishing up
```
Comment (PhilippeTurcotteGov) [inline guides/tfvc-to-git.md]:
```suggestion
### Optional Improvements
```
We could separate the optional improvements and the additional references material into distinct sections.

Comment (PhilippeTurcotteGov) [inline guides/tfvc-to-git.md]:
```suggestion
### Optional Improvements
```
We could separate the optional improvements and the additional references material into distinct sections.

File: guides/tfvc-to-git.md [hunk 2]
```diff
@@ -6,9 +6,33 @@
...
+
+* Robust your repository
+  * Add Git [Templates files](https://github.com/esdc-edsc/template-gabarit) to the Repository
+  * Add labels with the [ESDC Label Generator](https://github.com/esdc-edsc/label-generator) (for GCcode & GitHub)
+* Learn about Git
```
Comment (PhilippeTurcotteGov) [inline guides/tfvc-to-git.md]:
```suggestion

### Further Reading
* Learn about Git
```
We could separate the optional improvements and the additional references material into distinct sections.

File: guides/tfvc-to-git.md (hunk 1 above)
Comment (s-laugh) [reply guides/tfvc-to-git.md]: resolving as duplicate suggestion

Comment (s-laugh) [reply guides/tfvc-to-git.md]: Updated suggestion to `Suggested`.

File: guides/tfvc-to-git.md (hunk 2 above)
Comment (s-laugh) [reply guides/tfvc-to-git.md]: Updated suggestion to fix formatting of MD.

**Pull request merged**